### PR TITLE
New case studies section 

### LINKED ISSUE #201: new: case studies page
We'll have an index page with all case studies. Here are the redirections for the links on the related cards:
- **Finn.auto:** https://www.checklyhq.com/case-studies/finn-auto
- **SchwarzIT:** 
   Read Case Study: https://www.hashicorp.com/blog/monitoring-as-code-with-terraform-cloud-and-checkly
   Watch Webinar Video: https://www.youtube.com/watch?v=KPdCxMqprdc&t=2s
- **1Password:** https://blog.1password.com/securely-monitoring-scim-bridges/
Screen Shot 2021-09-21 at 19 45 39

Design: https://www.figma.com/file/et2KLDMVOPb0xeAmXFJXok/?node-id=2249%3A10807

### PR DESCRIPTION
**Pull request by pilimartinez**
Resolves #201 

Created a new "case studies" section that shows how our clients use Checkly.

On the [Figma](https://www.figma.com/file/et2KLDMVOPb0xeAmXFJXok/?node-id=2249%3A10807) mockup for mobile, there's a "Free trial" card at the end that is the one that we are currently using on the website, but tablet and desktop have a different and new one (check the attached screenshot for reference). I used the same one that's currently on the site for the 3 screen sizes but let me know if a new one is needed. 

Captura de Pantalla 2021-09-24 a la(s) 16 29 48

### CODE REVIEW

File: src/scss/style.scss [hunk 1]
```diff
@@ -32,6 +32,7 @@
 @import 'case-studies';
 @import 'about';
 @import 'careers';
+@import 'case-studies-main';
```
Comment (ianaya89) [inline src/scss/style.scss:35]: Can we rename to `case-studies`?

File: src/scss/_case-studies-main.scss [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      background-image: url("../casestudiesmain/finnauto.webp");
+    }
+    .locus {
+      background-image: url("../casestudiesmain/locus.webp");
+    }
+    .schwarz {
+      background-image: url("../casestudiesmain/schwarz.webp");
+    }
+    .password {
+      background-image: url("../casestudiesmain/1pass.webp");
```
Comment (ianaya89) [inline src/scss/_case-studies-main.scss:95]: This images directory should be `case-studies`

**Review by ianaya89 — CHANGES_REQUESTED** [review]
Looks good, just some minor changes

File: src/scss/style.scss (hunk 1 above)
Comment (pilimartinez) [reply src/scss/style.scss]: Done :)

File: src/scss/_case-studies-main.scss (hunk 2 above)
Comment (pilimartinez) [reply src/scss/_case-studies-main.scss]: Done :) Now both case studies main index and finn.auto index are on the same folder (case-studies)

**Review by ianaya89 — DISMISSED** [review]

**Review by ianaya89 — DISMISSED** [review]

**Comment by ianaya89** [comment]
💯 
@orcunilbeyli are we going to add a footer menu item or any kind of navigation to access this page? I think it is missing right now.

**Review by ianaya89 — APPROVED** [review]

**Pull request merged**